### PR TITLE
Smalls additions (see commits) in accordance with informant feedback

### PR DESCRIPTION
**Pull request by chhoumann**

### CODE REVIEW

File: Models/AssetComparer.cs [hunk 1]
```diff
@@ -59,14 +64,31 @@ private void UpdateAssetStates(IEnumerable<T> intersectingAssets)
         {
             foreach (T currentAsset in currentAssets)
             {
-                if (!intersectingAssets.Any(asset => asset.AssetId == currentAsset.AssetId))
+                T foundIntersectingAsset =
```
Comment (humleflue) [inline Models/AssetComparer.cs:63]: Mangler indrykning

**Review by humleflue — CHANGES_REQUESTED** [review]
LGTM - der er bare den ene indrykningsting der

File: Models/AssetComparer.cs (hunk 1 above)
Comment (chhoumann) [reply Models/AssetComparer.cs:63]: Should be OK in the code. Could be a Github thing

Comment (humleflue) [reply Models/AssetComparer.cs:63]: Never mind my bad

**Review by humleflue — APPROVED** [review]
Very niiice

**Pull request merged**